### PR TITLE
Runtime detection in copy_from_pqclean

### PR DESCRIPTION
**Pull request by baentsch**
This is the 'generator' PR as discussed with @dstebila for more easy reading/understanding. The 'fulltest' variant (after running `copy_from_pqclean.py`) is also available at https://github.com/baentsch/liboqs/tree/copy-pqclean-v3-fulltest (CCI test run results at https://circleci.com/gh/baentsch/liboqs/tree/copy-pqclean-v3-fulltest).

I'd be glad for an initial review and statement as to whether you're OK converting this to a PR (plus the information who shall run the actual copy operation).

### CODE REVIEW

**Comment by baentsch** [comment]
PS: AppVeyor also passes on the fulltest (after running `copy_from_pqclean`: https://ci.appveyor.com/project/MichaelBaentsch/liboqs/builds/31984036

File: tests/speed_sig.c [hunk 1]
```diff
@@ -43,6 +43,8 @@ static OQS_STATUS sig_speed_wrapper(const char *method_name, uint64_t duration,
 	printf("%-30s | %10s | %14s | %15s | %10s | %25s | %10s\n", sig->method_name, "", "", "", "", "", "");
 	TIME_OPERATION_SECONDS(OQS_SIG_keypair(sig, public_key, secret_key), "keypair", duration)
 	TIME_OPERATION_SECONDS(OQS_SIG_sign(sig, signature, &signature_len, message, message_len, secret_key), "sign", duration)
+//message[0]=0;
+//	printf("verify result = %d\n", OQS_SIG_verify(sig, message, message_len, signature, signature_len, public_key));
```
Comment (dstebila) [inline tests/speed_sig.c:47]: Some unused code?

**Comment by dstebila** [comment]
Looks good to me!  Okay with me if you want to convert to PR and do the full copy.

File: tests/speed_sig.c (hunk 1 above)
Comment (baentsch) [reply tests/speed_sig.c:47]: Ouch - Yes. Sorry. Will be removed in full PR. Brace yourself. Thanks for the quick feedback.

**Pull request merged**